### ISSUE TITLE
Guess who's back?

### ISSUE THREAD

**Issue by gsscoder** [gsscoder/commandline#406]
For the ones who have missed the cause of my last absence from OS/etc, please follow this link:

https://twitter.com/gsscoder/status/821821285701451777

Here everything is explained.

Bye :)

**Comment by nemec** [comment]
Welcome back @gsscoder!

**Comment by gsscoder** [comment]
Thanks! I think that having given full commit access to CommandLine main contributors allows the project to grow also with my coordination. Now you know the codebase better than me! ;)

Ciao

**Comment by dirkrombauts** [comment]
Does that mean there'll be an official 2.x release on nuget soon? That would be cool!

**Comment by gsscoder** [comment]
Yes, time allowing, but it will be in (advanced) beta status... :)
